tests/modules/dataplex_autodq/examples/datascan_test_inputs.tfvars

```diff
@@ -0,0 +1,105 @@
+# Copyright 2023 Google LLC
```

Comment:
I think this file is not used.

Reply:
I think I am missing the tftest.yaml file to turn this into a test according to this guide https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#testing-via-tfvars-and-yaml-aka-tftest-based-tests. Will look into adding this

Reply:
Indeed. If you want to run a test with this tfvars you'll have to create a tftest.yaml. Ping me if you need some help with it.

Reply:
This is completed.